APMrover2/test.cpp

```diff
@@ -214,8 +214,8 @@ int8_t Rover::test_wp(uint8_t argc, const Menu::arg *argv)
 {
     delay(1000);
 
-    cliSerial->printf("%u waypoints\n", (unsigned)mission.num_commands());
-    cliSerial->printf("Hit radius: %f\n", (double)g.waypoint_radius.get());
+    cliSerial->printf("%u waypoints\n", static_cast<unsigned int>(mission.num_commands()));
```

Comment:
these c++ casts are actually super ugly. It's ok when it's in objects, but on scalar types, not.

unsigned and unsigned int are the same type.